.gitignore

```diff
@@ -6,6 +6,8 @@
 build/
 DerivedData/
 
+Examples/Konstantin/
```

Comment:
Probably not needed :)

Reply:
I've created my own folder for some staff which shouldn't be pushed (like token and universe name) - but forgot to remove it in Keys class - so pushed anyway :-)

Reply:
Fair. Could you call it something that is not your name, like `LocalConfiguration` or something? Then we could all do it that way :) Not a biggie if not.